spec/factories/speakers.rb

```diff
@@ -21,4 +18,11 @@
   trait :with_empty_links do
     links { {} }
   end
+
+  trait :stub_http_ok do
+    after(:build) do |_speaker|
+      allow(HTTParty).to receive(:head).and_return(double(code: 200))
+      allow(HTTParty).to receive(:get).and_return(double(code: 200))
+    end
```

Comment:
Important step to decouple tests from external HTTP dependencies. Good call adding it here.